.github/workflows/test-environment.yml

```diff
@@ -229,6 +268,7 @@ jobs:
           terraform apply --auto-approve \
             -var="deployment_name=${{ env.DEPLOYMENT_NAME }}" \
             -var="region=${{ env.AWS_REGION }}" \
+            -var="ec_url=${{ env.TF_VAR_ec_url }}" \
```

Comment:
All variables following the `TF_VAR_name` pattern are [environment variables](https://developer.hashicorp.com/terraform/cli/config/environment-variables#tf_var_name) and should not be provided in the command; this one can be removed.

Reply:
@gurevichdmitry When we were testing the apply terraform deployment we had to update ec_url with the qa cloud url `terraform apply  -target "module.ec_project" -var='ec_url=https://console.qa.cld.elstc.co' -var='ess_region=aws-eu-west-1' -var='serverless_mode=true'`. If we remove the `ec_url` variable, then how can we create deployments in QA cloud environments?

Reply:
We have multiple options to assign variables to [root module variables](https://developer.hashicorp.com/terraform/language/values/variables#assigning-values-to-root-module-variables). The `-var` `ec_url` (command-line variable) is the same as `TF_VAR_ec_url` (environment variable), which is defined [here](https://github.com/elastic/cloudbeat/pull/1982/files#diff-e92acdebda21024508b591a7dae2cd202f1e96f7741c94ca643a88fbf9905666R133). By assigning beforehand, Terraform will know to use the environment variable for ec_url.

Reply:
I'll remove in that case.